### ISSUE TITLE
Tuple syntax no longer allows / to be used

### ISSUE THREAD

**Issue by annevk** [WICG/cookie-store#222]
Infra dropped this quite a while ago, but it seems to be still used here for cookie attributes?

**Comment by inexorabletash** [comment]
We must have missed a memo. 😉  

This can easily be changed (PRs welcome?); ultimately this is populating a [cookie-attribute-list](https://httpwg.org/http-extensions/draft-ietf-httpbis-rfc6265bis.html#section-5.6) which is a list of attributes, each one with an attribute-name and attribute-value. We approximate that with a list of lists, list of tuples, ordered map of key/value, define new concept types, etc. I don't have a strong preference.

**Comment by annevk** [comment]
Yeah, I'm not immediately PR'ing as Johann and I are also looking at changes needed to 6265bis so the contract might well change a bit.
